README.md

```diff
@@ -83,7 +83,7 @@ verificarlo works as expected on your system:
    $ make installcheck
 ```
 
-If you disable dragonegg support during configure, fortran_test will fail.
+If you disable dragonegg support during configure, fortran_test will be disable and be consider as passing the test.
```

Comment:
could you please change to "be disable**d** and considered as passing the test."

Reply:
of course ! Sorry...